src-plugins/itkFilters/itkMorphologicalFiltersToolBox.cpp

```diff
@@ -85,6 +85,7 @@ itkMorphologicalFiltersToolBox::itkMorphologicalFiltersToolBox ( QWidget *parent
     QLabel * morphoFilterLabel = new QLabel ( tr ( "Kernel radius:" ) );
     QHBoxLayout * morphoFilterLayout = new QHBoxLayout;
     d->mmButton = new QRadioButton(tr("mm"), this);
+    d->mmButton->setObjectName("mm");
```

Comment:
When finding the child you specify that it's a qPushButton (or a qLabel) and pb solved. In my case : 
```c++
getToolBox("itkMorphologicalFiltersToolBox").findChild<QPushButton*>("Run")->click();
```
Now to be truly honest (and I shouldn't because I would like to see this PR merged quickly :p), I see the pb coming with this method : 
```c++
step.hideToolBoxWidget("itkMorphologicalFiltersToolBox", "Run");
```
Now it's too "Friday evening" for me to start thinking about it, and I really liked the simplicity of what's currently done. So let's keep it warm for next week.